frontend/src/redux/Pokemon/__tests__/selectors.test.ts

```diff
@@ -0,0 +1,22 @@
+import { state } from '__fixtures__/state';
+import { getPokemon, getPokemons } from '../selectors';
+
+const pokemonState = {
+  1: { id: 1, name: 'bulbasaur', height: 7, weight: 69 },
+  2: { id: 2, name: 'ivysaur', height: 10, weight: 130 },
+};
+const initialState = { ...state, login: { token: null, loginError: null }, pokemon: pokemonState };
+
+describe('Pokemon selectors', () => {
+  describe('getPokemon function', () => {
+    it('Should return the first pokemon stored in store.pokemon', () => {
+      const pokemon = { id: 1, name: 'bulbasaur', height: 7, weight: 69 };
+      expect(getPokemon(initialState)).toStrictEqual(pokemon);
+    });
+  });
+  describe('getPokemons function', () => {
+    it('Should return an array of pokemons stored in store.pokemons', () => {
+      expect(getPokemons(initialState)).toStrictEqual(Object.values(pokemonState));
```

Comment:
You have rewritten your selector `Object.values(pokemonState)`, in order to avoid rewritting your function in your test, we prefer to realize some dumb comparison